DQM/SiPixelPhase1Common/src/SiPixelCoordinates.cc

```diff
@@ -0,0 +1,916 @@
+// -*- C++ -*-
```

Comment:
Update: I fail to get `SiPixelCoordinates` compiled in the `DataFormats/SiPixelDetId` package:

    /cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_9_0_X_2017-01-24-2300/src/DataFormats/CLHEP/interface/AlgebraicObjects.h:8:33: fatal error: CLHEP/Matrix/Vector.h: No such file or directory

Which is transitively included from `SiPixelCoordinates`, but worked in the DQM package. I do't see what is going on there...